### PR TITLE
Adding documentation to README and adding backend logic

### PR DESCRIPTION
**Pull request by serinamarie**
Description:
Adding prerequisites, getting started section, schema

Type of change:
Documentation update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,103 @@
...
 
-#### Request
+**Description**
 
-    GET 
+Returns the last name, first name, enroll date, exit date, income at entry, income at exit, and exit destination of a user in database.
 
-#### Response
-```json
+**POST Request**
```
Comment (guillearria) [inline README.md]: Example request and response JSON objects are cut off when you look at document in markdown. Try adding json or yaml after the triple back ticks, see: 

https://stackoverflow.com/questions/14901245/how-to-style-a-json-block-in-github-wiki

File: README.md [hunk 2]
```diff
@@ -1,27 +1,103 @@
...
+    ],
+    "ssn": [
+    integer, 
+    integer
+    ],
+    "pwd": string
+}
+```
 
+**Response**
```
Comment (guillearria) [inline README.md:73]: Same as above here

**Review by guillearria — CHANGES_REQUESTED** [review]
Looks good! Love the architecture diagram and clear development instructions. PR is ready to merge after updating the response and request examples. :rocket:

File: README.md (hunk 1 above)
Comment (serinamarie) [reply README.md]: That makes sense, someday I will be the master of markdown.

File: README.md (hunk 2 above)
Comment (serinamarie) [reply README.md:73]: I am clearly still a young markdown grasshopper.

**Comment by serinamarie** [comment]
I have made the requested changes. Please review once more!

**Comment by serinamarie** [comment]
I have edited the request and response bodies so they should not create those red error blocks. Instead of using json I used typescript docstrings.

**Comment by guillearria** [comment]
Looks great :fire:

**Pull request merged**